packages/commonwealth/server/routing/router.ts

```diff
@@ -370,6 +372,21 @@ function setupRouter(
     getRelatedCommunitiesHandler.bind(this, serverControllers),
   );
 
+  registerRoute(
+    router,
+    'get',
+    '/communityStakes/:community_id/:stake_id',
+    getCommunityStakeHandler.bind(this, models, serverControllers),
+  );
+
+  registerRoute(
+    router,
+    'put',
+    '/communityStakes/:community_id/:stake_id',
+    passport.authenticate('jwt', { session: false }),
+    putCommunityStakeHandler.bind(this, models, serverControllers),
```

Comment:
Who's authorized to create stakes?

Reply:
Thanks for pointing it out. I will add in the admin only check